src/tests/access/test_ownable.cairo

```diff
@@ -1,24 +1,16 @@
+use option::OptionTrait;
 use starknet::ContractAddress;
 use starknet::contract_address_const;
 use starknet::testing;
 use zeroable::Zeroable;
 use openzeppelin::access::ownable::Ownable;
-use openzeppelin::access::ownable::Ownable::OwnableImpl;
-use openzeppelin::access::ownable::Ownable::OwnableCamelOnlyImpl;
 use openzeppelin::access::ownable::Ownable::InternalImpl;
+use openzeppelin::access::ownable::Ownable::OwnableCamelOnlyImpl;
```

Comment:
Based on https://github.com/OpenZeppelin/cairo-contracts/pull/679#discussion_r1279645993, shouldn't `ownable::Ownable;` come after `ownable::Ownable::x;`? Again, the way it currently is set is the way it should be 🙃

Reply:
Yes :). Good catch. The formatter should fix the imports after using the right version though, I don't think we should focus that much on that.